test/tests.ml

```diff
@@ -234,3 +234,105 @@ let%expect_test "comparison" =
     comparison result: -1
     comparison result: 0
     comparison result: 1 |}]
+
+(* The differnces between [Pp.paragraph], [Pp.text], [Pp.verbatim] and box +
+   [Pp.text] when inside a [Pp.vbox]. *)
+let%expect_test "paragraph" =
+  let lorem =
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum \
+     euismod, nisl eget aliquam ultricies."
+  in
+  let seper text _ =
```

Comment:
What's a seper?

Reply:
Thats a visual seperator for seeing which output is which.. Ill give it a better name.

Reply:
Okay, `separator` reads easier.